hack/generate-operator-bundle-contents.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+# vim: ts=4:sw=4:cc=99
+
```

Comment:
I would add the source of this script in case it drifts

```suggestion
# Vendored from https://github.com/app-sre/deployment-validation-operator/blob/1a11c74c9f65f790fe963b18eb56a75396057d61/hack/generate-operator-bundle-contents.py
```